docs/CreateMessage/create-message.plantuml

```diff
@@ -0,0 +1,91 @@
+@startuml create-message
+!pragma teoz true
+
+title Create Message
+
+actor sender as "Sender"
+control fn as "CreateMessage"
+database db as "messages"
+collections st as "processing-message"
+participant ai as "ai-common"
+queue q as "message-created"
+
+sender -> fn ++: //Submit Message//
+
+fn -> fn: Parse the //Request Payload//
+
+note left
+The //Request Payload// includes
+the sender metadata
+and the //Message Payload//.
+end note
+
+break Invalid //Request Payload//
+  fn --x sender: Bad Request
+end
+
+fn -> fn: Parse the //FiscalCode// \nfrom //Request Body// or //Request Path//
+
+break Invalid //FiscalCode//
+  fn --x sender: Bad request
+end
+
+opt //Remoted Content//
+  break Invalid //configuration_id//
+    fn --x sender: Forbidden
+  end
+end
```

Comment:
Here we should have an API call to the `getRCConfiguration`, do we want to avoid it to simplify the diagram? The status we return to the caller varies based on the response from this call.